src/style.css

```diff
@@ -0,0 +1,95 @@
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.flex_container {
+  width: auto;
+  margin: 0;
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.flex_item {
+  font: 100px Arial;
+  color: white;
+
+  height: 300px;
+  background-color: red;
+
+  flex-basis: 300px;
+  flex-grow: 1;
+
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.flex_item:nth-child(1) {
```

Comment:
don't repeat yourself

Reply:
this justify-content is for text inside flex_item

Reply:
or what do you mean?

Reply:
I meant the color styling

Reply:
ah, okay, didn't see it, just misprinted it